### PR TITLE
Make documentation more complete

### PR DESCRIPTION
**Pull request by kongo09**
## Proposed change

The Nuki integration has some specifics that are not obvious just from the lock domain perspective. Especially using the lock and unlock services for the ring-to-open function was not documented. Also, the binary sensor has been added.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nuki.markdown [hunk 1]
```diff
@@ -25,8 +25,36 @@ To add a Nuki bridge to your installation, you need to enable developer mode on
 
 {% include integrations/config_flow.md %}
 
+## Binary Sensors
+
+The integration provides a binary sensor if the Nuki door sensor is available. 
+
 ## Services
 
+### Service `lock.unlock`
```
Comment (frenck) [inline source/_integrations/nuki.markdown:34]: These services do not belong to the Nuki integration but are part of the Lock component of Home Assistant. It should not be documented here, it is documented with the Lock integration documentation.

**Comment by frenck** [comment]
Hi there @kongo09 👋 

Thanks for willing (and the effort) to contribute, however, this is not how we document these services. So, unfortunately, I have to friendly decline this contribution.

../Frenck

**Comment by kongo09** [comment]
Yes, I noticed that pattern.

Can you please advise what the proper way then is to document the specifics of the Nuki integration? The lock component doesn't mention the "ring-to-open" feature of an opener. It also doesn't mention the sensor. What's the best way to make this clear to the user? I'm happy to adapt the contribution.

File: source/_integrations/nuki.markdown (hunk 1 above)
Comment (kongo09) [reply source/_integrations/nuki.markdown:34]: Is the binary sensor also part of the lock component? It is not documented there.

**Review by kongo09 — COMMENTED** [review]
@frenck can you please point me in the right direction on how to document the specifics of an integration? I studied the developer docs but they don't explain how to do this. I'm happy to adapt and also happy to add to the developer docs.

**Pull request closed without merging**